string cmdType

# float32 vx_r_front  # 在机器人坐标系下前侧的速度
# float32 vy_r_front
# float32 vx_r_back   # 在机器人坐标系下后侧的速度
# float32 vy_r_back

float32 vx_r        # 在机器人坐标系下的速度
float32 vy_r

bool tight_front  # 前侧的夹紧
bool tight_back   # 后侧的夹紧
float32 robot_kink_angle # 机器人弯折角度
# float32 x
# float32 y
# float32 dx
# float32 dy
# float32 fan_1
# float32 fan_2
# float32 fan_3
# float32 robot_vel
# float32[4] joystick_vel_w
# float32 polish_speed
# float32 per_x
# float32 per_y
# float32 num_track
# float32 orien_track